subprojects/core/src/integTest/groovy/org/gradle/api/tasks/NestedInputIntegrationTest.groovy

```diff
@@ -538,7 +538,7 @@ class NestedInputIntegrationTest extends AbstractIntegrationSpec implements Buil
         when:
         withBuildCache().run "customTask", "--info"
         then:
-        output.contains "Could not append invalid inputPropertyHash for 'bean.\$\$implementation\$\$' to build cache key."
+        output.contains "The implementation of 'bean.\$\$implementation' cannot be determined, because it was loaded via some unknown classloader"
```

Comment:
Use `an` instead of the too familiar `some`, and lose the `.$$implementation` part of the property name.

Reply:
done.